user-guide/exporting/index.njk

```diff
@@ -0,0 +1,29 @@
+---
+title: 08· Exporting
+---
+
+<h1 id="exporting">Exporting</h1>
+<p class="main-paragraph">At Penpot you can set export for objects or groups of them.</p>
+
+<p class="advice">Importing files and multiple exports are among the main priorities of the Penpot team. Related features are coming soon.</p>
+
+<h3 id="export">How to export</h3>
+<p>To export an object you need to select it and at the Design panel add an export preset pressing the “+” button of the Export section.</p>
+<p>Export presets can be also found at the <strong><a href=”/user-guide/02-the-interface/#viewmode-interface” target=”_blank”>View mode</a></strong> with the code tab activated.</p>
```

Comment:
Sections no longer have the 02- prefix in the slug. And I think that internal links should not have target="_blank".